menu.py

```
@@ -30,18 +32,62 @@ def _add_toolbar_entry() -> None:
+    except Exception:
+        return False
+
+
+def _run_oklch_init_for_node(node) -> None:
+    if not _is_oklch_grade_group(node):
+        return
+    try:
+        import oklch_grade_init
+        importlib.reload(oklch_grade_init)
+        oklch_grade_init.initialize_node(node)
+    except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)